src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int ballNumber = random.nextInt(100); // Generate Random Number for ball
```

Comment:
100 is a magic number. Add it as constant field MAX_BALL_NUMBER

![image](https://github.com/mate-academy/jv-lottery/assets/20326858/08c83571-bb5e-45ca-a5d0-fab07f8c47ee)